### PR TITLE
Some more packaging fixes before publishing a new version

### PR DESCRIPTION
**Pull request by doubleface**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,11 +19,13 @@
     "build"
   ],
   "scripts": {
-    "start": "tsc && node ./src/index.js",
-    "dev": "tsc && cozy-run-dev",
-    "standalone": "tsc && cozy-run-standalone",
+    "start": "node ./src/index.js",
```
Comment (laedit) [inline package.json]: Is this because you don't like typescript or just a mistake? 😄

Comment (doubleface) [reply package.json]: A mistake of course! Thanks for the review

**Comment by laedit** [comment]
@doubleface it seems that konnitor is based on the konnector-template for the checks so I have deleted the extra deploy on prod branch.
Is that ok?

**Comment by doubleface** [comment]
Yes it is ok. When the store will be deployed, only.the version in registry will be used

**Pull request merged**